homeassistant/components/group/cover.py

```diff
@@ -200,6 +200,13 @@ def current_cover_tilt_position(self):
         """Return current tilt position for all covers."""
         return self._tilt_position
 
+    @property
+    def state_attributes(self):
+        """Return the state attributes for the cover group."""
+        data = getattr(super(), "state_attributes", {})
+        data[ATTR_ENTITY_ID] = self._entities
```

Comment:
Do we have a precedence for using this attribute name for this purpose? In the person integration we use `source` attribute to mark the entity that provides the state. Is `entity_id` ambiguous to what entity it's referring to?

Reply:
Yes. There's a precedence in `group`.
https://github.com/home-assistant/core/blob/dev/homeassistant/components/group/__init__.py#L467

Reply:
Ok.